packages/core/src/modules/cache/singleContextLruCache/CacheRecord.ts

```diff
@@ -0,0 +1,41 @@
+import type { TagsBase } from '../../../storage/BaseRecord'
+
+import { BaseRecord } from '../../../storage/BaseRecord'
+import { uuid } from '../../../utils/uuid'
+
+export interface CacheItem {
+  value: unknown
+  expiresAt?: number
+}
+
+export interface CacheStorageProps {
+  id?: string
+  createdAt?: Date
+  tags?: TagsBase
+
+  entries: Array<{ key: string; item: CacheItem }>
```

Comment:
Woudn't a `Map` make more sense here?

Reply:
This is the structure as used by lru_map. Can change it to a map, however that would be a breaking change to the existing cache records (I wanted to avoid breaking changes for now)

Reply:
Fixed. It's now a map